### ISSUE TITLE
Startup Parameters screen enhancements

### ISSUE THREAD

**Issue by pedrotski** [wisp-gg/issue-tracker#25]
### Describe the bug

Links in tool tips are not wrapped or clickable. You also cannot select text from a tooltip.

https://imgur.com/a/aad3N3K

https://github.com/wisp-gg/frontend/issues/426

### Expected Behavior

Allow clickable URL's (with underline and your accent color) in the tooltips. This would be made better if we could use some whitelisted html in the description to add anchor text.

### Actual Behavior

You can't click or copy anything.

### Steps to Reproduce

na

### Browser Details

na

### Operating System

na

**Comment by tcatlas** [comment]
Honestly, I'd like to see this page completely revamped to serve as a way for hosting providers to create a well-organized settings page for all server settings for games.

**Comment by pedrotski** [comment]
> Honestly, I'd like to see this page completely revamped to serve as a way for hosting providers to create a well-organized settings page for all server settings for games.

Agree, some kind of tab features could be set if a field was added to variables to group them together. 

Example, if I had a variable with the name Gameplay, it would generate that tab and group them all on a new page. That way the client isnt looking at 100 different settings on one screen.

This was more of a bug report of something that i feel is critical to fix as a stop gap until the redesign. For example, hosts could link their guides into the tool tip to give the user a better explanation of what the setting goes.
